packages/react/src/components/TextInput/Root/index.tsx

```diff
@@ -0,0 +1,58 @@
+import { ComponentProps, createContext, useContext } from 'react'
+import { VariantProps, tv } from 'tailwind-variants'
+
+const input = tv(
+  {
+    base: [
+      'flex w-60 flex-1 flex-row items-center justify-between rounded-md border border-neutral-50 bg-white font-sf-heading text-neutral-80',
+      'focus-within:border focus-within:border-primary-green-500',
+    ],
+
+    variants: {
+      size: {
+        sm: 'px-4 py-2.5 text-sf-sm',
+        md: 'px-4 py-2.5 text-sf-md',
+        lg: 'px-5 py-3.5 text-sf-lg',
+      },
+      disabled: {
```

Comment:
do we still need this variant?

Reply:
because this looks the case of a validation using the prop inline and tailwind merge

Reply:
mmm, we had a convo about this and concluded that keeping the disabled as a variant is a cleaner implementation